### DISCUSSION TITLE
Help setting up

### DISCUSSION THREAD

**Discussion by HotSurfSpot** [groue/GRDB.swift#1090] [Q&A] [answered]
Hey guys,

I'm trying to set up GRDB for my app. I've installed the pod but there is something I am doing wrong when trying to create my database connection.

I have tried writing these lines in my App.swift file:

```

import GRDB

let dbQueue = try DatabaseQueue(path: "/Desktop/App")
let inMemoryDBQueue = DatabaseQueue()
```

But an error occurs and it says: 

Call can throw, but errors cannot be thrown out of a global variable initializer

What am I doing wrong ?
Also, I read in the documentation that when writing these lines, a .sqlite file should be created in the folder given in the path but that is not the case.

Thanks !

**Comment by StuClift** [comment, marked as answer]
> Call can throw, but errors cannot be thrown out of a global variable initializer

Creating the Database file can fail but you are not defining what should happen if it does. I'd recommend reading the [Error Handling](https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html#) section of the Swift language manual. 

If you want to just get something to compile then you can use `try!` 

Also, `"/Desktop/App"` is almost certainly not going to create the file you expect. Take a look at one of the [examples](https://github.com/groue/GRDB.swift/blob/master/Documentation/DemoApps/GRDBDemoiOS/GRDBDemoiOS/Persistence.swift).
